pluribus/games/short_deck/state.py

```diff
@@ -332,7 +332,7 @@ def legal_actions(self) -> List[Optional[str]]:
         actions: List[Optional[str]] = []
         if self.current_player.is_active:
             actions += ["fold", "call"]
-            if self._n_raises < 3 or self._poker_engine.n_active_players == 2:
+            if self._n_raises < 3:
```

Comment:
Removing this condition as when there are only two active players, this allows for an infinite string of raises, as a raise is then always a legal action, and the betting round is never complete.